apps/prairielearn/elements/pl-file-editor/pl-file-editor.py

```diff
@@ -152,7 +192,7 @@ def render(element_html, data):
 
     if data["panel"] == "question":
         html_params["question"] = True
-        with open("pl-file-editor.mustache", "r", encoding="utf-8") as f:
+        with open("pl-ace-static.mustache", "r", encoding="utf-8") as f:
```

Comment:
I think this may be leftover from a different course-specific element, can you check?

Reply:
Yes, I seemed to have accidentally edited this in the source file instead of the file in the personal course.  I made sure to change it back.